lox_services/utils/chrome_version.py

```diff
@@ -0,0 +1,78 @@
+import os
+import re
+from sys import platform
+
+from lox_services.utils.general_python import print_info
+
+def extract_version_registry(output):
+    """Extracts the version of Chrome from the registry output.
+
+    Args:
+        output (str): The output of the registry query.
+    """
+    try:
+        google_version = ''
+        for letter in output[output.rindex('DisplayVersion    REG_SZ') + 24:]:
+            if letter != '\n':
+                google_version += letter
+            else:
+                break
+        return google_version.strip()
+    except TypeError:
+        return
+
+def extract_version_folder():
```

Comment:
missing return type

Reply:
is this function only for windows, if yes good to precise it in the function name and documentation